### PR TITLE
Use the name of the viewElement as the shortName.

### PR DESCRIPTION
**Pull request by Natim**
r? @vladikoff

### CODE REVIEW

File: src/heading.js [hunk 1]
```diff
@@ -86,10 +86,18 @@ export default class Heading extends Plugin {
...
-				return options[ index ] ? options[ index ].title : defaultTitle;
+			  const index = areActive.findIndex( value => value );
+
+			  // If none of the commands is active, display default title.
+              let title = defaultTitle;
+              if (options[index]) {
+                if (options[index].viewElement) {
+                  title = options[index].viewElement.toUpperCase();
+                } else {
+                  title = "P";
```
Comment (vladikoff) [inline src/heading.js:97]: nit: do we need to refactor this?

**Review by vladikoff — APPROVED** [review]
Looks good!

File: src/heading.js (hunk 1 above)
Comment (Natim) [reply src/heading.js:97]: What do you mean?

Comment (vladikoff) [reply src/heading.js:97]: I think Ryan wanted to have this say `Aa` or `aA`, @ryanfeeley do you remember?

Comment (Natim) [reply src/heading.js:97]: This code uses Aa when nothing is selected P, H3, H2, H1 when something is. However my previous commit was using Aa all the time.

**Pull request merged**